src/Microsoft.AspNet.Mvc.Core/Logging/ScopeValues.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public class ScopeValues : LoggerStructureBase
+    {
+        public ScopeValues([NotNull] string scopeName)
+        {
+            ScopeName = scopeName;
+        }
+
+        public string ScopeName { get; }
```

Comment:
The design of this it too generic. We don't want 'name' we want to include an identifier related to the action. Probably need to add a guid to each action, and then include that and the action display name.